docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine
+
+MAINTAINER Anand Tamariya <atamariya@gmail.com>
+
+RUN apk add --no-cache ffmpeg mediainfo openjdk7-jre 
+
+WORKDIR /usr/src/ums
+
+VOLUME /media
+
+VOLUME /profile
+
+EXPOSE 9001 5002 1044
+
+CMD java -DUMS_PROFILE=/profile -Dfile.encoding=UTF-8 -Djava.net.preferIPv4Stack=true -Djna.nosys=true -cp ums.jar net.pms.PMS 
```

Comment:
Does it need memory instructions like 1280mb?

Reply:
Added JVM_OPTS environment variable for runtime memory setting. However, memory allotted to container will also play an important role. https://github.com/akashche/docs/blob/4325be5ccb52721bfa26521beaee40e607c7b7af/openjdk/content.md

Reply:
Ok great, thanks. That looks like a fine default. We have historically used 768mb for 32-bit and 1280mb for 64-bit systems. I don't know enough about docker to properly review this PR but I can do some reading about it, or do you think it is just safe to merge now?

Reply:
It's safe to merge.